release/ray_release/scripts/ray_bisect.py

```diff
@@ -59,12 +74,17 @@ def _bisect(test: Test, commit_list: List[str], concurrency: int) -> str:
         idx_to_commit = {}
         for i in range(concurrency):
             idx = len(commit_list) * (i + 1) // (concurrency + 1)
+            # make sure that idx is not at the boundary; this avoids rerun bisect
+            # on the previously run revision
+            idx = min(max(idx, 1), len(commit_list) - 2)
```

Comment:
could you change this to a more readable `if.. then` statement? like 

````
if idx < 1:
   idx = 1
if idx >= len(commit_list) - 1:
   idx = len(commit_list) - 2
````

or maybe we should just skip it?

maybe consider make this a function that returns a list of index like `def _bisect_index(n: int, concurrency: int) -> List<int>` and have a test for that?

Reply:
wait w00t this code is already in master, this will go away as I rebase. I can re-work it in another PR